packages/toolpad-app/src/toolpad/Toolpad.tsx

```diff
@@ -58,12 +59,14 @@ export default function Toolpad({ basename }: EditorProps) {
       <Box sx={{ height: '1px', minHeight: '100vh' }}>
         <ErrorBoundary fallbackRender={({ error }) => <FullPageError error={error} />}>
           <React.Suspense fallback={<FullPageLoader />}>
-            <BrowserRouter basename={basename}>
-              <Routes>
-                <Route path="/" element={<Home />} />
-                <Route path="/app/:appId/*" element={<AppWorkspace />} />
-              </Routes>
-            </BrowserRouter>
+            <UserFeedback>
```

Comment:
i didn't immediately understand this was a context provider from the component name - maybe we could make it more explicit?